semantic-ui/index.d.ts

```diff
@@ -0,0 +1,5434 @@
+// Type definitions for Semantic UI 2.2
+// Project: http://semantic-ui.com/
+// Definitions by: Leonard Thieu <https://github.com/leonard-thieu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="jquery" />
+
+// tslint:disable:unified-signatures
+
+interface JQuery {
+    // TODO: Do all modules have a 'setting' behavior?
+
+    // region Modules
+
+    // region Accordion
+
+    /**
+     * Refreshes all cached selectors and data
+     */
+    accordion(behavior: 'refresh'): JQuery;
+    /**
+     * Opens accordion content at index
+     */
+    accordion(behavior: 'open', index: number): JQuery;
+    /**
+     * Closes accordion content that are not active
+     */
+    accordion(behavior: 'close others'): JQuery;
+    /**
+     * Closes accordion content at index
+     */
+    accordion(behavior: 'close', index: number): JQuery;
+    /**
+     * Toggles accordion content at index
+     */
+    accordion(behavior: 'toggle', index: number): JQuery;
+    accordion(settings: SemanticUI.AccordionSettings): JQuery;
+    accordion(): JQuery;
+
+    // endregion
+
+    // region Checkbox
+
+    /**
+     * Switches a checkbox from current state
+     */
+    checkbox(behavior: 'toggle'): JQuery;
+    /**
+     * Set a checkbox state to checked
+     */
+    checkbox(behavior: 'check'): JQuery;
+    /**
+     * Set a checkbox state to unchecked
+     */
+    checkbox(behavior: 'uncheck'): JQuery;
```

Comment:
Each of these overloads should be collapsed into a union type as follows:

```ts
checkbox(behavior: "toggle" | "check" | "uncheck")
```

The same goes for the overloads below if they don't return the same object.